airflow/providers/amazon/aws/operators/neptune.py

```diff
@@ -32,6 +36,49 @@
     from airflow.utils.context import Context
 
 
+def handle_waitable_exception(
+    operator: NeptuneStartDbClusterOperator | NeptuneStopDbClusterOperator, err: str
+):
+    """Handle client exceptions for invalid cluster or invalid instance status that are temporary.
```

Comment:
```suggestion
    """
    Handle client exceptions for invalid cluster or invalid instance status that are temporary.
```

Non-blocking:   I forget which rule it is offhand, but one of the `ruff` rules we are trying to turn on is that multi-line docstrings like this start on a new line.  Someone will have to come back and change this later, may as well do it now if you don't mind.